kofta/src/app/components/UserProfile.tsx

```diff
@@ -175,6 +176,7 @@ export const UserProfile: React.FC<UserProfileProps> = ({
             </div>
           );
         })}
+        Playing {useRPC()}
```

Comment:
Won;t this make it so the RPC only works for the user that has it on? Others wont be able to see it i assume

Reply:
@TheOtterlord said the same thing. Isn't this component visible to everyone. Which is the visible UserProfile?

Reply:
It is visible to everyone, but the data is coming from the user's computer, not the backend, so even if they look at someone elses account, it will show their RPC. You need some sort of storage on the backend to deal with this. This along with the new design I believe is a reason to wait for now, and plan out how this would work

Reply:
Yes @TheOtterlord is right. I would say use the method that is being used to update the user's bio rn and make it a part of their bio

Reply:
Oh ok